src/main/resources/db/changelog/changes/06-create-categories-table.yaml

```diff
@@ -0,0 +1,29 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-categories-table
+      author: vadymhrnk
+      changes:
+        - createTable:
+            tableName: categories
+            columns:
+              - column:
+                  name: id
+                  type: bigint
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
```

Comment:
It is not necessary to add nullable: false if you have a primaryKey

Reply:
agree